src/k8s/cmd/k8s/k8s_set.go

```diff
@@ -66,119 +65,69 @@ func newSetCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
-	case "ingress.enable-proxy-protocol":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for ingress.enable-proxy-protocol: %w", err)
-		}
-		config.Ingress.EnableProxyProtocol = &v
-	case "local-storage.enabled":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for local-storage.enabled: %w", err)
-		}
-		config.LocalStorage.Enabled = &v
-	case "local-storage.local-path":
-		config.LocalStorage.LocalPath = utils.Pointer(value)
-	case "local-storage.reclaim-policy":
-		config.LocalStorage.ReclaimPolicy = utils.Pointer(value)
-	case "local-storage.default":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for local-storage.default: %w", err)
-		}
-		config.LocalStorage.Default = &v
-	case "load-balancer.enabled":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for load-balancer.enabled: %w", err)
-		}
-		config.LoadBalancer.Enabled = &v
-	case "load-balancer.cidrs":
-		config.LoadBalancer.CIDRs = utils.Pointer(strings.FieldsFunc(value, func(r rune) bool { return unicode.IsSpace(r) || r == ',' }))
-	case "load-balancer.l2-mode":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for load-balancer.l2-mode: %w", err)
-		}
-		config.LoadBalancer.L2Mode = &v
-	case "load-balancer.l2-interfaces":
-		config.LoadBalancer.L2Interfaces = utils.Pointer(strings.FieldsFunc(value, func(r rune) bool { return unicode.IsSpace(r) || r == ',' }))
-	case "load-balancer.bgp-mode":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for load-balancer.bgp-mode: %w", err)
-		}
-		config.LoadBalancer.BGPMode = &v
-	case "load-balancer.bgp-local-asn":
-		v, err := strconv.Atoi(value)
-		if err != nil {
-			return fmt.Errorf("invalid integer value for load-balancer.bgp-local-asn: %w", err)
-		}
-		config.LoadBalancer.BGPLocalASN = &v
-	case "load-balancer.bgp-peer-address":
-		config.LoadBalancer.BGPPeerAddress = utils.Pointer(value)
-	case "load-balancer.bgp-peer-port":
-		v, err := strconv.Atoi(value)
-		if err != nil {
-			return fmt.Errorf("invalid integer value for load-balancer.bgp-peer-port: %w", err)
-		}
-		config.LoadBalancer.BGPPeerPort = &v
-	case "load-balancer.bgp-peer-asn":
-		v, err := strconv.Atoi(value)
-		if err != nil {
-			return fmt.Errorf("invalid integer value for load-balancer.bgp-peer-asn: %w", err)
-		}
-		config.LoadBalancer.BGPPeerASN = &v
-	case "metrics-server.enabled":
-		v, err := strconv.ParseBool(value)
-		if err != nil {
-			return fmt.Errorf("invalid boolean value for metrics-server.enabled: %w", err)
-		}
-		config.MetricsServer.Enabled = &v
-	default:
-		return fmt.Errorf("unknown config key %q", key)
+	if _, ok := knownSetKeys[key]; !ok {
+		return fmt.Errorf("unknown option key %q", key)
```

Comment:
Should we add "consult `k8s get` command to see configurable options to the error message?